Submitted By: Joe Ciccone <jciccone@gmail.com>
Date: 2008-02-03
Rediffed and changed on 2008-07-13
Initial Package Version: 2.6.23
Upstream Status: Not Submitted
Origin: Joe Ciccone
Description: Define PAGE_SIZE outside of ifdef __KERNEL__ so it's not removed
             when the headers are sanatized.
             We also need the linux/const.h for _AC

diff -Naur linux-2.6.24.7.orig/include/asm-alpha/page.h linux-2.6.24.7/include/asm-alpha/page.h
--- linux-2.6.24.7.orig/include/asm-alpha/page.h	2008-05-06 19:22:34.000000000 -0400
+++ linux-2.6.24.7/include/asm-alpha/page.h	2008-07-13 18:24:31.000000000 -0400
@@ -1,15 +1,16 @@
 #ifndef _ALPHA_PAGE_H
 #define _ALPHA_PAGE_H
 
-#ifdef __KERNEL__
-
 #include <linux/const.h>
-#include <asm/pal.h>
 
 /* PAGE_SHIFT determines the page size */
-#define PAGE_SHIFT	13
-#define PAGE_SIZE	(_AC(1,UL) << PAGE_SHIFT)
-#define PAGE_MASK	(~(PAGE_SIZE-1))
+#define PAGE_SHIFT      13
+#define PAGE_SIZE       (_AC(1,UL) << PAGE_SHIFT)
+#define PAGE_MASK       (~(PAGE_SIZE-1))
+
+#ifdef __KERNEL__
+
+#include <asm/pal.h>
 
 #ifndef __ASSEMBLY__
 
